SpatialGDK/Documentation/content/non-unreal-server-worker-types.md

```diff
@@ -1,75 +1,85 @@
+* `Worker_RequestId SendCommandRequest(Worker_EntityId EntityId, const Worker_CommandRequest* Request, uint32_t CommandId);`
+* `void SendCommandResponse(Worker_RequestId RequestId, const Worker_CommandResponse* Response);`
+
+To receive [network operations](https://docs.improbable.io/reference/latest/shared/design/operations) for external SpatialOS components, you must provide custom callbacks for specific component IDs and operation types via `USpatialDispatcher` functions. These functions are parameterized with a `Worker_ComponentId` and a callback function const reference that takes a different network operation types as an argument:
+
+* `OnAddComponent` parameterized with `const TFunction<void(const Worker_AddComponentOp&)>&`
+* `OnRemoveComponent` parameterized with `const TFunction<void(const Worker_RemoveComponentOp&)>&`
+* `OnAuthorityChange` parameterized with `const TFunction<void(const Worker_AuthorityChangeOp&)>&`
+* `OnComponentUpdate` parameterized with `const TFunction<void(const Worker_ComponentUpdateOp&)>&`
+* `OnCommandRequest` parameterized with `const TFunction<void(const Worker_CommandRequestOp&)>&`
+* `OnCommandResponse` parameterized with `const TFunction<void(const Worker_CommandResponseOp&)>&`
 
-If you want to ensure that the SpatialOS worker connection receives your callback for initial network operations, you need to register the callbacks inside your game instance's `OnConnected` event callback.
+If you want to ensure that the SpatialOS worker connection registers your callbacks to receive the initial network operations, you need to register the callbacks inside your game instance's `OnConnected` event callback.
 
-Each `USpatialDispatcher::AddOpCallback` function returns a `CallbackId`. You can deregister your callbacks using the `USpatialDispatcher::RemoveOpCallback` function and passing the `CallbackId` parameter that was returned by the corresponding call to `USpatialDispatcher::AddOpCallback`.
+Each callback registration function returns a `CallbackId`. You can deregister your callbacks using the `USpatialDispatcher::RemoveOpCallback` function and passing the `CallbackId` parameter that was returned by the corresponding call to one of the callback registration functions.
 
-There is a basic example in the _Examples_ section below. For more examples of how to deserialize the `Worker_Op` type see the SpatialOS documentation on [serialization in the Worker SDK in C’s API](https://docs.improbable.io/reference/latest/capi/serialization).
+There are basic example in the _Examples_ section below. For more examples of how to serialize and deserialize component updates, command requests, and more, see the SpatialOS documentation on  [serialization in the Worker SDK in C’s API](https://docs.improbable.io/reference/latest/capi/serialization). `Worker_ComponentId` and each network operation type are defined in the [Worker SDK in C’s API](https://docs.improbable.io/reference/latest/capi/api-reference).
```

Comment:
Convert plural to singular or vice versa.

Reply:
thanks updated